contentcuration/contentcuration/tests/testdata.py

```diff
@@ -200,7 +241,40 @@ def create_temp_file(filebytes, preset='document', ext='pdf', original_filename=
                               original_filename=original_filename,
                               file_on_disk=storage_file_path)
 
-    return {'name': os.path.basename(storage_file_path), 'data': filebytes, 'file': fileobj, 'db_file': db_file_obj}
+    return {
+        'name': os.path.basename(storage_file_path),
+        'data': filebytes,
+        'file': fileobj,
+        'db_file': db_file_obj
+    }
+
+
+def create_test_file(filebytes, ext='pdf'):
+    """
+    Create a temporary file with contents of `filebytes` for use in tests.
+    :param filebytes: The data to be stored in the file (as bytes)
+    :param ext: File extension, omitting the initial period
+    Returns a dict containing the following:
+    - checksum (str): md5 hash of file contents
+    - name (str): the filename within the content storage system (= checksum + . + ext )
+    - storagepath (str): the relative storage path for this file storage/c/h/checksum.ext
+    - data (bytes): file content (echo of `filebytes`)
+    - file (file): an instance of TemporaryFile object that you can read/write
+    """
+    hash = hashlib.md5(filebytes)
+    checksum = hash.hexdigest()
+    filename = "{}.{}".format(checksum, ext)
+    storage_file_path = cc.generate_object_storage_name(checksum, filename)
+    fileobj = TemporaryFile()
+    fileobj.write(filebytes)
+    fileobj.seek(0)
+    return {
+        'checksum': checksum,
+        'name': os.path.basename(storage_file_path),
+        'storagepath': storage_file_path,
+        'data': filebytes,
+        'file': fileobj
+    }
```

Comment:
This is a new one -- temp file but do not save file in storage. This will be used for ricecooker tests.